AWSIoT/AWSIoTDataManager.h

```diff
@@ -417,15 +417,27 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)enableMetricsCollection:(BOOL)enabled;
 
 /**
+ @deprecated This method is deprecated and will be deleted in the next minor version. Please use `updateUserMetaData` instead.
  Set user-specified dictionary of the additional values to be passed as components of
  connection username.
  *Swift*
  let userMetaData: [String: String] = ["AFRSDK": "ios", "AFRSDKVersion": "1.0.0", "AFRLibVersion":"1.4.1"]
  iotDataManager.addUserMetaData(userMetaData)
  @param userMetaData A dictionary of key-value metadata pairs to be appended to the connection username
  */
+DEPRECATED_MSG_ATTRIBUTE("Use `updateUserMetaData` for updating the user meta data");
```

Comment:
I think this syntax is incorrect. `DEPRECATED_MSG_ATTRIBUTE` goes after the method declaration, before the terminating semicolon: `- (void)addUserMetaData:(NSDictionary<NSString *, NSString *> *)userMetaData DEPRECATED_MSG_ATTRIBUTE ("Use `updateUserMetaData` for updating the user meta data");`

Either way, please check this locally and make sure that Xcode properly reports the method as deprecated at the call site.

Reply:
Verified that Xcode flags the deprecation warning upon call to ```addUserMetaData``` method